front/app/containers/Admin/projects/project/analysis/Tasks/index.tsx

```diff
@@ -1,66 +1,167 @@
-import React from 'react';
+import React, { useState } from 'react';
 import useAnalysisBackgroundTasks from 'api/analysis_background_tasks/useAnalysisBackgroundTasks';
 import { useParams } from 'react-router-dom';
 import {
-  Accordion,
   Box,
-  Spinner,
   colors,
+  IconButton,
+  Dropdown,
+  Button,
+  Text,
 } from '@citizenlab/cl2-component-library';
 import ProgressBar from 'components/UI/ProgressBar';
 import styled from 'styled-components';
+import { useIntl } from 'utils/cl-intl';
+import Divider from 'components/admin/Divider';
+import { TagTypeColorMap } from '../Tags/Tag';
 
 const StyledProgressBar = styled(ProgressBar)`
   height: 8px;
   width: 100%;
 `;
 
+const stateColorMap = {
+  in_progress: colors.blue500,
+  queued: colors.brown,
+  succeeded: colors.success,
+  failed: colors.error,
+};
+
 const Tasks = () => {
+  const [isDropdownOpened, setIsDropdownOpened] = useState(false);
+
   const { analysisId } = useParams() as { analysisId: string };
   const { data: tasks } = useAnalysisBackgroundTasks(analysisId);
 
+  const { formatTime, formatDate } = useIntl();
+
   const anythingInProgress = tasks?.data.find(
     (t) =>
       t.attributes.state === 'in_progress' || t.attributes.state === 'queued'
   );
 
   return (
-    <Box bg="white" mb="8px">
-      <Accordion
-        title={
-          <Box p="24px">
-            {anythingInProgress ? (
-              <Spinner size="20px" />
-            ) : (
-              'Recent background jobs'
-            )}
+    <Box display="flex" w="32px" h="32px">
+      {anythingInProgress ? (
+        <Button
+          processing={true}
+          buttonStyle="text"
+          width="24px"
+          height="24px"
+          aria-label="background jobs"
+        />
+      ) : (
+        <IconButton
+          iconName="book"
```

Comment:
It's... weird :)
Best to ask Rob for something suitable

Reply:
Agree - will ask him for a better icon, we might need to introduce a new one..